earth_enterprise/rpms/build.gradle

```diff
@@ -685,7 +716,7 @@ task openGeeFusionRpm (type: GeeRpm, dependsOn: openGeeFusionInitScripts) {
 
     packageName = 'opengee-fusion'
     version = ospackage.version
-    release = "1.${rpmPlatformString}"
+    release = ospackage.release
```

Comment:
where is this coming from now, as release is more specifically about rpm build/package revision...

Reply:
It is the build number (i.e. what is returned as part of `earth_enterprise/src/scons/getversion.py -l`).  They do say that the main intent is for packaging but then also mention it can be used for hot fixing which in a sense is what we are doing by putting the build number in there  Also, we don't have separate repositories/build process for packaging the binaries vs building the Open GEE binaries and so where not using this at all (just hard coding it to `1`).  Because of all this it seems okay to hijack it for the purposes of distinguishing what build within a release cycle this package was created from.  It does bend the rules a bit but gives us the behavior we need/desire.